### ISSUE TITLE
Patch Error

### ISSUE THREAD

**Issue by tuca1234567890** [e666666/robloxWineBuildGuide#16]
When I run `git apply roblox-wine-staging-v2.2.patch` it gives me this error:
```
error: patch failed: wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg:260
error: wine-tkg-git/wine-tkg-profiles/advanced-customization.cfg: patch does not apply
```

I'm on Arch Linux.

**Comment by karimrir1** [comment]
Fixed by commit bc57508ad7412f2e8699715292c9b25b6698f067.

**Comment by liu3631** [comment]
> Fixed by commit [bc57508](https://github.com/e666666/robloxWineBuildGuide/commit/bc57508ad7412f2e8699715292c9b25b6698f067).

I think this is unrelated to the OP's report. I'm also having the issue on Arch even if I'm using the latest patch from the latest commit. Looks like the line numbering of advanced-customization.cfg file from the wine-tkg source code changed, therefore the patch doesn't match with the line 260.

**Comment by tuca1234567890** [comment]
> > Fixed by commit [bc57508](https://github.com/e666666/robloxWineBuildGuide/commit/bc57508ad7412f2e8699715292c9b25b6698f067).
> 
> I think this is unrelated to the OP's report. I'm also having the issue on Arch even if I'm using the latest patch from the latest commit. Looks like the line numbering of advanced-customization.cfg file from the wine-tkg source code changed, therefore the patch doesn't match with the line 260.

yeah, I "fixed" it by deleting the line from the patch file that conflicts with advanced-customization. just tried it now

**Comment by karimrir1** [comment]
Alright, it should work now.
